.github/workflows/triage-issues.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow uses the following github action to automate
+# management of stale issues and prs in this repo:
+# https://github.com/marketplace/actions/close-stale-issues
+
+name: Close stale issues and PRs
+
+on: 
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+  
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 10
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          stale-issue-label: auto-triage-stale
+          stale-issue-message: 👋 It looks like this issue has been open for 30 days with no activity. We'll mark this as stale for now, and wait 10 days for an update or for further comment before closing this issue out.
+          close-issue-message: As this issue has been inactive for more than one month, we will be closing it. Thanks again for writing in!
+          exempt-issue-labels: auto-triage-skip
+          exempt-all-milestones: true
+          remove-stale-when-updated: true
+          enable-statistics: true
+          debug-only: true # TODO: remove after test run
```

Comment:
@srajiang when to remove this line?

Reply:
Very shortly - I'm reviewing the output of the test run now. If all clear on that and it's doing what I think it should be doing, we can remove it. I won't include this line for PRs in other repos.

Reply:
The output is here: https://github.com/slackapi/bolt-js/runs/4257717061?check_suite_focus=true in case you'd like to review the triage results (when performed in debug mode).